style.css

```diff
@@ -1,22 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;1,300;1,400;1,500&display=swap");
+
 * {
   margin: 0;
 }
 
+body {
+  font-family: "Exo", sans-serif;
+  background-color: #fff;
+  background-image: linear-gradient(315deg, #fff 100%, #d7e1ec 74%);
+}
+
 main {
   padding: 10px;
+  font-family: "Poppins", sans-serif;
 }
 
 .book-item {
-  padding-top: 10px;
-  padding-bottom: 10px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
 }
 
-#form-add-book {
-  padding-top: 10px;
+.book-item:nth-child(odd) {
+  background-color: #dcdcdc;
+}
+
+.book-item:nth-child(even) {
+  background-color: #fff;
 }
 
 header {
   border: 3px solid black;
   padding: 10px 20px;
   align-self: stretch;
 }
+
+.heading {
+  margin-bottom: 20px;
+  font-family: "Poppins", sans-serif;
+}
+
+#books-dynamic-container {
+  border: 3px solid black;
+  box-shadow: 5px 2px 2px rgb(0 0 0);
+}
+
+h2 {
```

Comment:
[Optional]
I'll recommend you to use CSS classes instead of directly styling the HTML elements because CSS classes are reusable and if you've to use the same HTML element with a different style then ultimately you've to use CSS classes then it's better to always start styling with CSS classes. 😄

Reply:
Thanks!